spec/schemas.coffee

```diff
@@ -21,7 +20,7 @@ describe 'Schemas', ->
   schemas.forEach (schema) ->
     describe "#{schema.id} (#{schema.title})", ->
       try
-        cases = lib.getExamples schemaName
+        cases = lib.getExamples path.basename schema.id, path.extname schema.id
```

Comment:
`getExamples` works on names only (`contentblock`), while `schema.id` is full filename (`contentblock.json`)